go/client/cmd_status.go

```diff
@@ -199,7 +199,7 @@ func (c *CmdStatus) outputJSON(status *fstatus) error {
 func (c *CmdStatus) outputTerminal(status *fstatus) error {
 	dui := c.G().UI.GetDumbOutputUI()
 	dui.Printf("Username:      %s\n", status.Username)
-	dui.Printf("Logged in:     %s\n", BoolString(status.LoggedInProvisioned, "yes", "no"))
+	dui.Printf("Logged in:     %s\n", BoolString(status.LoggedInProvisioned && status.PassphraseStreamCached, "yes", "no"))
```

Comment:
Maybe we could just not change the status output so we can see that everything is working in the wild (logged in and passphrase stream cached both always yes).